src/main/java/com/jfrog/ide/idea/scan/ScanManager.java

```diff
@@ -23,11 +24,13 @@
 import java.util.Set;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
 
 import static com.jfrog.ide.common.log.Utils.logError;
 import static com.jfrog.ide.common.utils.XrayConnectionUtils.createXrayClientBuilder;
 
 public class ScanManager {
+    private final long SCAN_TIMEOUT_MINUTES = 10;
```

Comment:
int requires less memory
```suggestion
    private final int SCAN_TIMEOUT_MINUTES = 10;
```